src/provider/provider_tracking_linux.c

```diff
@@ -0,0 +1,25 @@
+/*
+ *
+ * Copyright (C) 2024 Intel Corporation
+ *
+ * Under the Apache License v2.0 with LLVM Exceptions. See LICENSE.TXT.
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ *
+ */
+
+#include "critnib.h"
+#include "provider_tracking.h"
+
+#include <stddef.h>
+
+critnib *TRACKER = NULL;
```

Comment:
nit: this var can be static I think

Reply:
Oh, I've changed the Windows file, forgot about this one. Thanks, now it's done.